.ent.el

```diff
@@ -0,0 +1,29 @@
+;;; .ent.el --- local ent config file -*- lexical-binding: t; -*-
```

Comment:
this file probably should not be committed :)

Reply:
You are right :)